docs/app/Examples/elements/List/Content/ListDescriptionExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListDescriptionExample extends Component {
+  render() {
+    const mapIcon = <i className='map marker icon' />;
+
+    return (
+      <List>
+        <ListItem image={mapIcon} header='Chicago' description='This city is located in the state of Illinois' />
+        <ListItem image={mapIcon} header='Nashville' description='This city is located in the state of Tennessee' />
```

Comment:
Use the `icon` prop to pass in an icon.  The `image` prop is for passing in a constructed `<Image />`.  I'll add an issue for validation on this.